### PR TITLE
Limit box style + sass format

### PR DESCRIPTION
**Pull request by sroy3**
screen shot 2017-10-16 at 9 55 50 am

Still waiting on a new color for gray labels since they're a bit hard to read.

### CODE REVIEW

**Comment by sroy3** [comment]
Label color was updated 

screen shot 2017-10-16 at 10 47 25 am

File: docs/_components/components/19_limit_box.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    <span class="limit-box-usage-value">100</span>
+                </div>
+                <div class="limit-box-limit form-group input-field validate">
+                    <input id="SecondLimit" type="number" value="110" required />
+                    <label for="SecondLimit">Limit</label>
+                </div>
+            </div>
+        </div>
+        <div class="limit-box-footer">
+            <div class="limit-box-bar progress-91"></div>
```
Comment (GermainBergeron) [inline docs/_components/components/19_limit_box.html]: The progress-x is always an integer?

File: scss/components/limit-box.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@for $i from 0 through 100 {
+  .limit-box-bar.progress-#{$i} {
+    width: $i * 1%;
+
+    @if $i >= 75 {
+      background-color: $orange;
+    }
+    @if $i == 100 {
+      background-color: $red;
+    }
```
Comment (GermainBergeron) [inline scss/components/limit-box.scss]: I wonder if we should use `mods` instead. In some cases maybe 100 is better than 0

File: docs/_components/components/19_limit_box.html (hunk 1 above)
Comment (sroy3) [reply docs/_components/components/19_limit_box.html]: You fill it up with the rounded percentage. I should have added an example without a bar also, but it's possible to have a box without usage and without a progress bar.

File: scss/components/limit-box.scss (hunk 2 above)
Comment (sroy3) [reply scss/components/limit-box.scss]: That's the way I did it at first, but it felt like more work for the user to add the mod. With a mod you have to chose when to use them. This way, there are default values, but you can always change them the same way you would have with a mod. I could add 3 mods to easily overwrite the color.

Comment (sroy3) [reply scss/components/limit-box.scss]: I added the mods and more examples.

**Comment by sroy3** [comment]
Looks like my latest commit isn't shown here. Here is what I added: https://github.com/coveo/vapor/commit/3428413e618dbf71a203400060e3d0c50f59f78e
I'll make sure everything is there when merging and releasing.

**Pull request merged**